tests/devtools.test.tsx

```diff
@@ -148,26 +148,6 @@ describe('If there is no extension installed...', () => {
       createStore(devtools(() => ({ count: 0 })))
     }).not.toThrow()
   })
```

Comment:
I mean the test itself is fine.

```suggestion
  })
  
    it('does not warn', async () => {
    createStore(devtools(() => ({ count: 0 })))
    expect(console.warn).not.toBeCalled()
  })
```

Reply:
@charkour I do not have a permission to add a commit.

Reply:
Sorry I misunderstood. I'll make it editable. Thank you.

Reply:
Oh, I found out that my personal organization to store forks means I cannot make it editable. 

I've updated the name of the test as requested. Thank you